tests/test_cli/test_inputhandler/test_inputhandler.cpp

```diff
@@ -345,22 +347,12 @@ TEST(JsonFileHandler, writeOutputFile)
     jsonHandler.writeDataToFile(transaction.getSerializedTransaction());
 }
 
-TEST(PemFileReader, printFileContent)
-{
-    std::map<uint32_t, std::string> inputData;
-    inputData[ARGS_TX_IDX_PEM_INPUT_FILE] = "..//testData//keys.pem";
-    ih::wrapper::PemHandlerInputWrapper const pemWrapper(inputData);
-    ih::PemInputHandler pemHandler(pemWrapper);
-
-    pemHandler.printFileContent();
-}
-
 TEST(PemFileReader, isPemFileValid_validFile)
 {
     std::map<uint32_t, std::string> inputData;
     inputData[ARGS_TX_IDX_PEM_INPUT_FILE] = "..//testData//keys.pem";
     ih::wrapper::PemHandlerInputWrapper const pemWrapper(inputData);
-    ih::PemInputHandler pemHandler(pemWrapper);
+    ih::PemFileReader pemHandler(pemWrapper.getPemFilePath());
```

Comment:
👍  for the tests.

Reply:
Thank you, appreciate it ! :)
Although, we are missing a lot of unit tests for now.